public/less/themes/arc-green.less

```diff
@@ -85,6 +85,59 @@ body {
     color: #9e9e9e;
 }
 
+/* firefox scroll bars */
+
+html,
+body,
+* {
+    scrollbar-width: thin;
+    scrollbar-color: #87ab63 rgba(255,255,255,0.1);
```

Comment:
Maybe `scrollbar-color: #87ab63 transparent` and the same for `::-webkit-scrollbar-track` too?

Reply:
i don't know if this is a good idea.

just my 2 cents:

* `scrollbar-color: #87ab63 transparent`
  * pro: no changes to system theme scrollbar track. (Really better than my first aproach where it was set to `white` manually)
  * con: if mozilla changes behaviour and the second parameter styles the whole thumb, see webkit
* `::-webkit-scrollbar-track`
  * if i change this to transparent, the tracks can't be seen anymore on (desktop) webkit browsers. (mobile browsers mostly ignore track color and autohide thumb)
![grafik](https://user-images.githubusercontent.com/24939127/60144417-8a39bc80-97c2-11e9-8e4a-550963fbe912.png)
May you could not say there is even a scrollbar, as there is a green bullet only (realy long content)
  * I think that's weird.

Reply:
My primary concern is the full-white background on Firefox. I assume it's because the alpha channel of the color does not look to work. Chrome already looks good.

Reply:
@silverwind The full white should be fixed. The screenshot were taken before my request: https://github.com/go-gitea/gitea/pull/7269/commits/80eaef9aae3709bc2344769e2588faeee9c01e3c

Reply:
Ah, I see, thanks.

Reply:
@sapk thanks for screenshot